ui/page/sign_message_page.go

```diff
@@ -274,6 +274,12 @@ func (pg *SignMessagePage) Handle() {
 		clipboard.WriteOp{Text: pg.signedMessageLabel.Text}.Add(gtx.Ops)
 		pg.Toast.Notify("Signature copied")
 	}
+
+	if pg.infoButton.Button.Clicked() {
```

Comment:
Why this being handled here? The subpage already handles displaying the info modal https://github.com/planetdecred/godcr/blob/bd6fa8a89280d6a372414dbe8f7c97d249f5dd57/ui/page/sign_message_page.go#L116 

There're also similar changes in several pages.

Reply:
when enter key is pressed on Sign message info modal, verify message info modal, transaction details inffo modal and security tolls info modal, the app crashes. that was why this was implemented